sklearn/impute/_iterative.py

```diff
@@ -502,7 +503,7 @@ def _initial_imputation(self, X):
         Xt = X[:, valid_mask]
         mask_missing_values = mask_missing_values[:, valid_mask]
 
-        return Xt, X_filled, mask_missing_values
+        return Xt, X_filled, mask_missing_values, _missing_mask
```

Comment:
We need to update the docstring of `_initial_imputation`. The new return can be called `X_missing_mask`.